views/src/Pages/Mentordashboard.jsx

```diff
@@ -1,33 +1,56 @@
-import React from 'react';
-import Sidebar from '../components/Sidebar';
-import { Routes, Route, useLocation } from 'react-router-dom'; 
-import { IconListDetails, IconCheckupList, IconUser } from '@tabler/icons-react';
-import FeedbackRequestQueue from '../Pages/FeedbackQueue';
-import AssignedFeedback from '../Pages/AssignedFeedback';
-import Account from '../Pages/Account';
-import TopBar from '../components/TopBar';
+import React from "react";
+import Sidebar from "../components/Sidebar";
+import { Routes, Route, useLocation } from "react-router-dom";
+import {
+  IconListDetails,
+  IconCheckupList,
+  IconUser,
+  IconTable,
+} from "@tabler/icons-react";
+import FeedbackRequestQueue from "../Pages/FeedbackQueue";
+import AssignedFeedback from "../Pages/AssignedFeedback";
+import Account from "../Pages/Account";
+import TopBar from "../components/TopBar";
+import Checklist from "./Checklist";
 
 function Mentordashboard(props) {
   const navItems = [
-    { icon: IconListDetails, label: 'Feedback Queue', to: 'feedbackqueue' },
-    { icon: IconCheckupList, label: 'Assigned Feedback', to: 'assigned' },
-    { icon: IconUser, label: 'Account', to: 'account' }, 
+    { icon: IconListDetails, label: "Feedback Queue", to: "feedbackqueue" },
+    { icon: IconCheckupList, label: "Assigned Feedback", to: "assigned" },
+    { icon: IconUser, label: "Account", to: "account" },
+    { icon: IconTable, label: "Checklist", to: "checklist" },
   ];
 
-  const location = useLocation(); 
+  const location = useLocation();
 
   return (
-    <div style={{ display: "flex", flexDirection: "column-reverse", position: "relative" }}>
+    <div
+      style={{
```

Comment:
Instead of adding the styles directly to the element, why not create a style variable and feed that to the style property.
That way it's much cleaner.  so something like below

```
    const styleForElement = {
         display: "flex",
        flexDirection: "column-reverse",
        position: "relative",
    } 
    
    <h1 style={styleForElement }>My HEADING<h1>
```

Reply:
Will look at this.